frontend/src/components/journal.js

```diff
@@ -210,7 +218,7 @@ const Journal = ({ currentUser }) => {
   };
   
   const sortedExercises = exercises.sort((a, b) => {
-    const order = ['Running', 'Swimming', 'Cycling', 'Gym', 'Other'];
+    const order = ['Running', 'Swimming', 'Cycling', 'Gym', 'Walking', 'Other'];
```

Comment:
can we change this to:
`const order = ['Running', 'Swimming', 'Cycling', 'Walking', 'Gym', 'Other'];` just so that all exercise with stats appear first.

Reply:
Updated the ordering for Journal page 
![image](https://github.com/VathsalaAchar/MLA-pilot-group4/assets/22783570/55427315-9725-4b85-af4e-e7afedc5dfa3)